ansible/roles/common/tasks/main.yml

```diff
@@ -16,6 +16,12 @@
     line: '127.0.0.1 {{ liquid_domain.split(".")[0] }}'
     state: present
 
+- name: Put FQDN in /etc/hosts
+  lineinfile:
+    path: /etc/hosts
+    line: '127.0.0.1 {{ liquid_domain }}'
```

Comment:
Is this entry correct? Looking at my `/etc/hosts` file:

```
127.0.0.1	localhost
127.0.1.1	myhostname
```

So maybe the IP should be `127.0.1.1` and the hostname should be just the name, not the FQDN?

Reply:
I'm using `127.0.0.1` everywhere. Don't know why the debian default is like that. I think it shouldn't matter, as the whole `127.0.0.0/8` block is loopback.

We have both the hostname and the FQDN set up in the hosts. We need the FQDN for the oauth redirects.

Reply:
For the oauth login dance, I think we should have `/etc/resolv.conf` point to our own dns server.

Reply:
That dns server isn't always on. If you want to implement that, we need to have the server always on, so that's a separate issue.

Reply:
ok, [here](https://github.com/liquidinvestigations/liquidinvestigations/issues/90)m